views/newsletter-page.js

```diff
@@ -0,0 +1,83 @@
+
+$(document).ready(function(){
+	createNewsletterTemplate ();
+	enableSubmitClick ();
+})
+
+function createNewsletterTemplate () {
+	let template = $('#newsletter-page-template').html();
+	let compiledTemplate = Handlebars.compile(template);
+	$('#application').html(compiledTemplate());
+}
+
+function enableSubmitClick () {
+    let forms = document.getElementsByClassName('needs-validation');
+    let validation = Array.prototype.filter.call(forms, function(form) {
+    	let button = document.getElementById('button');
+		button.onclick = function () {
+			form.addEventListener('submit', function(event) {
+	        if (form.checkValidity() === false) {
+	          event.preventDefault();
+	          event.stopPropagation();
+	          console.log('sprobuj jeszcze raz')
+	        }
+	        form.classList.add('was-validated');
+	        event.preventDefault();
+	        savePersonalData ();
+	        form.innerHTML = '';
+	        document.getElementById('welcome-text').innerHTML = '';
+	        document.getElementById('thank-you-text').classList.remove('d-none')
+	        console.log('hurra')
+
+      	}, false);
+	 	}
+	});
+}
+
+function createThankYouTemplate () {
+	let template = $('#thank-you-template').html();
+	let compiledTemplate = Handlebars.compile(template);
+	$()
+}
+
+function savePersonalData () {
+	var forms = document.getElementsByClassName('needs-validation');
+	Array.from(forms).forEach(function(form){
+		if (form.classList.contains('was-validated') == true) {
+			console.log('grzybek')
+			const emailInput = document.getElementById('email-input');
+			const nameInput = document.getElementById('name-input');
+			const nappiesUserInput = document.getElementById('nappies-user-checkbox');
+			const producerInput = document.getElementById('producer-checkbox');
+			const shopOwnerInput = document.getElementById('shop-owner-checkbox')
+			let email = emailInput.value;
+			let name = nameInput.value;
+			let nappiesUser = false;
+			let producer = false;
+			let shopOwner = false;
+			if (nappiesUserInput.checked == true) {
+				nappiesUser = true;
+			};
+			if (producerInput.checked == true) {
+				producer = true;
+			};
+			if (shopOwnerInput.checked == true) {
+				shopOwner = true;
+			};
+			let dbRef = firebase.database().ref('newsletter/');
+			var newDbRef = dbRef.push();
+			newDbRef.set({
+			  	'email': email,
+				'name': name,
+				'nappiesUser': nappiesUser,
+				'producer': producer,
+				'shopOwner': shopOwner
+			});
```

Comment:
Hint: 
If you construct an object with field names that are the same as variable names that you assign to them, JavaScript provides an abbreviated notation for the assignment:
```
const name = 'john';
const email = 'john@gmail.com';
const someObject = {
  name,
  email
};
```
(So you could probably write that object in one line. ;))